test/1_stdlib/BridgeNonVerbatim.swift

```diff
@@ -33,7 +33,7 @@ import ObjectiveC
 var trackedCount = 0
 var nextTrackedSerialNumber = 0
 
-final class Tracked : ForwardIndex, CustomStringConvertible {
+final class Tracked : Equatable, CustomStringConvertible {
```

Comment:
Same here, you should be able to replace `Tracked` with `LifetimeTracked` and remove half of the code in the test.